src/server.cpp

```diff
@@ -3557,7 +3566,7 @@ bool Server::dynamicAddMedia(std::string filepath,
 	// Newer clients get asked to fetch the file (asynchronous)
 	pkt << token;
 	// Older clients have an awful hack that just throws the data at them
-	legacy_pkt.putLongString(filedata);
+	legacy_pkt.putLongString(*filedata);
```

Comment:
`filedata` is allowed to be `nullptr` in the header declaration, but here it seems it is assumed not to be `nullptr`.